packages/compiler/src/myst.ts

```diff
@@ -36,5 +59,95 @@ export function parsePanel(contents: string): Array<MystCard> {
     parsedCard.body = body;
     splitCards.push(parsedCard);
   }
-  return splitCards;
+
+  let myPanel: MystPanel = { cards: splitCards };
+  if (panelStyle) {
+    myPanel = { ...myPanel, style: panelStyle };
+  }
+  return myPanel;
+}
+
+export function parseStyling(contents: string): [string, string] {
```

Comment:
Could you rename this to something like `parseDirective`? I think that might be a little more clear.

Reply:
https://github.com/irydium/irydium/blob/ae9b5c1fc923c5c0f92b5ae15ed3782011f70ada/packages/compiler/src/myst.ts#L70